features/auth/signup/Signup.tsx

```diff
@@ -176,33 +166,6 @@ export default function Signup() {
     if (authState.error) window.scroll({ top: 0, behavior: "smooth" });
   }, [authState.error]);
 
-  useEffect(() => {
-    (async () => {
-      if (urlToken) {
-        setLoading(true);
-        try {
-          authActions.updateSignupState(
-            await service.auth.signupFlowEmailToken(urlToken)
-          );
-        } catch (err) {
-          Sentry.captureException(err, {
-            tags: {
-              component: "auth/signup/Signup",
-            },
-          });
-          authActions.authError(
-            isGrpcError(err) ? err.message : t("global:error.fatal_message")
-          );
-          router.push(signupRoute);
-          return;
-        }
-        setLoading(false);
-      }
-    })();
-    // next-router-mock router isn't memoized, so putting router in the dependencies
-    // causes infinite looping in tests
-  }, [urlToken, authActions, t]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
This is such an ugly usage of `useEffect` (it should have been managed by `useMutation`) that I'm glad we can get rid of it 😂